src/blob_gc_job.cc

```diff
@@ -26,6 +29,7 @@ class BlobGCJob::GarbageCollectionWriteCallback : public WriteCallback {
       fprintf(stderr, "GetImpl err, status:%s\n", s.ToString().c_str());
       abort();
     }
+    RecordTick(stats_, BLOB_DB_BYTES_READ, key_.size() + index_entry.size());
     if (s.IsNotFound()) {
       // Either the key is deleted or updated with a newer version which is
       // inlined in LSM.
```

Comment:
All the `Status::Busy` cases below should add to `BLOB_DB_GC_NUM_KEYS_OVERWRITTEN`. And the success case can log to `BLOB_DB_GC_NUM_KEYS_RELOCATED`. I think we can move the ticker counts to `RewriteValidKeyToLSM` by checking return code of `Write()`.

Reply:
the definition of `BLOB_DB_GC_NUM_KEYS_RELOCATED` is the number of keys dropped by garbage collection because they had been relocated to new blob file. It doesn't indicate the number of keys that have rewrited.  Should I change the definition?

Reply:
seems it is no need distinguishing the reason of dropped keys. I will change the definition.